promises.cf.in

```diff
@@ -331,11 +331,30 @@ bundle common cfengine_controls
 
 bundle common services_autorun
 # @brief Include autorun policy and discover autorun bundles if enabled
+# Directories defined in `def.mpf_extra_autorun_inputs` will have their
+# inputs automatically added as well (if the directory exists, if not it will be
+# silently skipped).
```

Comment:
> Directories will have their inputs automatically added

Directories will have their files added to inputs automatically

Or 

Files inside directories listed in `def.mpf_extra_autorun_inputs` will be added to inputs automatically.

Reply:
ACK